Source/WebCore/dom/Document+Fullscreen.idl

```diff
@@ -43,4 +43,11 @@
 
     [NotEnumerable] attribute EventHandler onwebkitfullscreenchange;
     [NotEnumerable] attribute EventHandler onwebkitfullscreenerror;
+
+    // Standard version.
```

Comment:
Give this is the standard one, I think you should swap things by putting this at the top of the interface, and annotating the non-standard ones as non-standard with the comments we usually add for such things.

Reply:
Makes sense, will do.